next/components/user-menu.tsx

```diff
@@ -72,8 +72,8 @@ export function UserMenu() {
     <div>
       <span className="sr-only">{t("user-menu")}</span>
       <button type="button" className="hover:underline" onClick={toggle}>
-        <span className="hidden sm:mr-2 sm:inline">
-          {t("user-menu-account")}
+        <span className="sr-only capitalize sm:not-sr-only sm:mr-2 sm:inline">
+          {t("log-in-or-register")}
```

Comment:
This "log-in-or-register" string has been changed in main branch (it's just "account" now)